include/crm/common/resources.h

```diff
@@ -378,8 +378,9 @@ typedef struct resource_object_functions_s {
      *
      * \return A node where the resource is active, preferring the source node
      *         if the resource is involved in a partial migration, or a clean,
-     *         online node if the resource's "requires" is "quorum" or
-     *         "nothing", otherwise NULL.
+     *         online node if the resource's \c PCMK_META_REQUIRES is
+     *         \c PCMK_VALUE_QUORUM or \c PCMK__VALUE_NOTHING, otherwise
+     *         \c NULL.
```

Comment:
This really belongs in the upcoming `PCMK_VALUE_QUORUM` commit. I can fix it (and will if there's other review), but it's only doxygen.